src/imitation/algorithms/adversarial.py

```diff
@@ -259,11 +269,17 @@ def train_gen(self, total_timesteps: Optional[int] = None, callback=None):
       callback: Callback argument to the Stable Baselines `RLModel.learn()`
         method.
     """
+    if self.error_on_unexpected_policy_change:
```

Comment:
Why does this check belong in `eval_disc_loss`?

Reply:
this is actually in `train_gen`